src/Category/PERM.agda

```diff
@@ -0,0 +1,65 @@
+{-# OPTIONS --type-in-type #-}
+
+module Category.PERM where
+
+open import Categories
+open Category
+open import Relation.Binary.PropositionalEquality as Eq using (_≡_; refl; cong; sym; trans)
+open Eq.≡-Reasoning using (begin_; _≡⟨⟩_; _∎; step-≡)
+open import Relation.Binary.Structures
+
+import Isomorphisms
+open Isomorphisms.Isomorphism
+open import Category.SET
+
+open import Isomorphisms SET
+
+
+record PermObj : Set where
+  field
+    permCarrier : Set
+    permAuto : Automorphism permCarrier
+
+open Isomorphism
+open PermObj
+
+record PermArrow ( A B : PermObj ) : Set where
+  field
+    permMap : permCarrier A → permCarrier B
+    permArrowLaw
+      : (a : permCarrier A)
+      → permMap (forward (permAuto A) a) ≡ forward (permAuto B) (permMap a)
+
+open PermArrow
+
+
+PERM : Category
+Obj     PERM = PermObj
+_~>_    PERM = PermArrow
+_≈_     PERM f g = forall a → permMap f a ≡ permMap g a
```

Comment:
I poked around for a bit in the stdlib for a good isEquivalence proof of this, but the situation right now for functions + setoids is an absolute disaster :(